packages/client/src/config.js

```diff
@@ -7,6 +7,7 @@ export default {
   store: null,
   reduxLogger: null,
   reduxStoreKey: 'accounts',
+  tokenStorage: global.localStorage,
   server: '',
   localStoragePrefix: '',
```

Comment:
Maybe rename `localStoragePrefix` to `tokenStoreagePrefix` to be more inline with the naming scheme.